airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java

```diff
@@ -46,13 +46,12 @@ public class BigQueryRecordConsumer extends FailureTrackingAirbyteMessageConsume
   private final boolean use1s1t;
   private final TyperDeduper typerDeduper;
 
-
   public BigQueryRecordConsumer(final BigQuery bigquery,
-      final Map<AirbyteStreamNameNamespacePair, AbstractBigQueryUploader<?>> uploaderMap,
-      final Consumer<AirbyteMessage> outputRecordCollector,
-      final String defaultDatasetId,
-      TyperDeduper typerDeduper,
-      final ParsedCatalog catalog) {
+                                final Map<AirbyteStreamNameNamespacePair, AbstractBigQueryUploader<?>> uploaderMap,
+                                final Consumer<AirbyteMessage> outputRecordCollector,
+                                final String defaultDatasetId,
+                                TyperDeduper typerDeduper,
+                                final ParsedCatalog catalog) {
     this.bigquery = bigquery;
```

Comment:
Fwiw, I don't agree with this format change. I prefer the 4-character indent, as a rule, because it is kinder towards not requireing rightward-scrolling, text wrapping, and violating common-sense line length limits.

Reply:
As if in a meta-commentary to prove my point, GitHub's diff is almost unreadable without horizontal scrolling here. 😅 

<img width="650" alt="image" src="https://github.com/airbytehq/airbyte/assets/18150651/251e4205-299d-40de-9009-c4076345c282">